openudm/CSVToolkit.cpp

```diff
@@ -2,31 +2,19 @@
 
 using namespace std;
 
-int GetIntFromString(std::string input) {
-	stringstream ss;
-	int i;
-	ss << input;
-	ss >> i;
-	return i;
+int GetIntFromString(const std::string& input) {
+	return std::stoi(input);
 }
 
-float GetFloatFromString(std::string input) {
-	stringstream ss;
-	float f;
-	ss << input;
-	ss >> f;
-	return f;
+float GetFloatFromString(const std::string& input) {
+	return std::stof(input);
 }
 
-double GetDoubleFromString(std::string input) {
-	stringstream ss;
-	double d;
-	ss << input;
-	ss >> d;
-	return d;
+double GetDoubleFromString(const std::string& input) {
+	return std::stod(input);
 }
 
-void ExtractCSV(std::string ipfile, int numItems, int extractItem, std::string* data) {
+void ExtractCSV(std::string ipfile, int numItems, int extractItem, std::vector<std::string>& data) {
```

Comment:
@virgesmith thanks for the PR!

@akademy comments look sound - for info, there's a +/- button here when making comments to "insert a suggestion" that renders as a quick-click-to-accept change:

```suggestion
void ExtractCSV(const std::string& ipfile, int numItems, int extractItem, std::vector<std::string>& data) {
```